### DISCUSSION TITLE
F´ Roadmap Released!

### DISCUSSION THREAD

**Discussion by LeStarch** [nasa/fprime#3082] [Announcements]
After a long wait, it is my pleasure to present to you, our beloved community, our F´  Roadmap.

Please see our roadmap: https://github.com/nasa/fprime/wiki/F%C2%B4%C2%A0-Roadmap

> F Prime plans and prioritizes on a rolling basis. We will do our best to keep that document up-to-date!

**Comment by gmarchetx** [comment]
Thank you for the roadmap, this is insightful. I recommend adding priorities in the document so that we, users, have an idea of when the items that are listed in that roadmap have a chance to be available. Rough estimates of dates would be ideal, but this is probably too demanding to keep up to date, hence priorities instead.
